torchvision/transforms/functional_tensor.py

```diff
@@ -3,12 +3,19 @@
 from torch.jit.annotations import List, BroadcastingList2
 
 
-def _is_tensor_a_torch_image(input):
-    return input.ndim >= 2
+@torch.jit.export
+def _is_tensor_a_torch_image(x: Tensor) -> bool:
+    return x.ndim >= 2
 
 
-def vflip(img):
-    # type: (Tensor) -> Tensor
+@torch.jit.export
```

Comment:
same comment here, we now have recursive compilation, so I don't think this is actually needed. Plus, I thought that `torch.jit.export` was needed only for decorating `nn.Module` methods?

Reply:
same here, i'll remove it